src/main/java/salesforce/ui/pages/lightning/contracts/NewContractPage.java

```diff
@@ -0,0 +1,274 @@
+package salesforce.ui.pages.lightning.contracts;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import salesforce.ui.pages.BasePage;
+import java.util.HashMap;
+
+public class NewContractPage extends BasePage {
+    @FindBy(css = "input[placeholder='Search Accounts...']")
+    private WebElement accountName;
+    @FindBy(xpath = "//mark[normalize-space()='TestAccount']")
+    private WebElement accountSelector;
+    @FindBy(css = "input[placeholder='Search Contacts...']")
+    private WebElement customerSignedBy;
+    @FindBy(xpath = "//div[@title='TestContact']")
+    private WebElement contactSelector;
+    @FindBy(xpath = "(//div/input[@class = ' input'])[2]")
+    private WebElement customerSignedTittle;
+    @FindBy(xpath = "(//div/input[@class = ' input'])[3]")
+    private WebElement customerSignedDate;
+    @FindBy(css = "input[placeholder='Search Price Books...']")
+    private WebElement priceBook;
+    @FindBy(xpath = "//mark[normalize-space()='Standard']")
+    private WebElement priceBookSelector;
+    @FindBy(xpath = "(//div/input[@class =' input'])[1]")
+    private WebElement contractStartDate;
+    @FindBy(css = ".input.uiInputSmartNumber")
+    private WebElement contractTermMonths;
+    @FindBy(css = ".select[aria-required='false']")
+    private WebElement ownerExpirationNotice;
+    @FindBy(linkText = "15 Days")
+    private static HashMap<String, String> ownerExpirationNoticeSelector = new HashMap<>();
+    @FindBy(xpath = "(//div/input[@class = ' input'])[4]")
+    private WebElement companySignedDate;
+    @FindBy(css = "textarea[placeholder='Billing Street']")
+    private WebElement billingStreet;
+    @FindBy(css = "input[placeholder='Billing City']")
+    private WebElement billingCity;
+    @FindBy(css = "input[placeholder='Billing State/Province']")
+    private WebElement billingState;
+    @FindBy(css = "input[placeholder='Billing Zip/Postal Code']")
+    private WebElement billingPostalCode;
+    @FindBy(css = "input[placeholder='Billing Country']")
+    private WebElement billingCountry;
+    @FindBy(xpath = "(//textarea[@class=\" textarea\" ])[1]")
+    private WebElement specialTerms;
+    @FindBy(xpath = "(//textarea[@class=\" textarea\" ])[last()]")
+    private WebElement description;
+    @FindBy(css = "button[title='Save']")
+    private WebElement save;
+    @FindBy(css = "button[title='Save & New']")
+    private WebElement saveAndNew;
+    @FindBy(css = "button[title='Cancel']")
+    private WebElement cancel;
+
+    static {
```

Comment:
Why did you create a map if the values will be equal to the keys

Reply:
It wa a really big mistake, I have already solved